lib/mods_display/fields/access_condition.rb

```diff
@@ -82,10 +82,23 @@ def copyright_statement(element)
     end
 
     def license_statement(element)
-      matches = element_text(element).match(/^(?<code>.*) (?<type>.*):(?<description>.*)$/)
+      element_text = element_text(element)
+      legacy_matches = element_text.match(/^(?<code>.*) (?<type>.*):(?<description>.*)$/)
 
-      return element_text(element) unless matches
+      return legacy_license_statement(element, legacy_matches) if legacy_matches
 
+      matches = element_text.match(/^This work is licensed under a (.+?)\.$/)
+
```

Comment:
I would remove these spaces so the return statements are visually associated with the previous lines.  :shrug:

Reply:
Done.